lib/core/app/controllers/concerns/ros/application_controller_concern.rb

```diff
@@ -80,7 +74,7 @@ def current_jwt
       end
 
       def auth_type
-        @auth_type ||= ActiveSupport::StringInquirer.new(request.env['HTTP_AUTHORIZATION'].split[0].downcase)
+        @auth_type ||= request.env['HTTP_AUTHORIZATION'].split[0].downcase
```

Comment:
I think it's generally preferable to use ```request.env.fetch('HTTP_AUTHORIZATION', "").split[0]&.downcase``` instead of having this blow up during tests if for whatever reason the variable is not set.

Reply:
thanks. i'll update that